app/components/Project/About/About.js

```diff
@@ -106,7 +108,37 @@ const about = props => {
       props.onAddedNote(props.project, text);
     }
   };
+  const handleCancel = () => {
+    if (unsavedChanges) {
+      const confirmCancel = dialog.showMessageBoxSync({
+        type: 'question',
+        buttons: ['Yes', 'No'],
+        title: 'Confirm Cancel',
+        message: 'You have unsaved changes. Do you wish to discard those changes?'
+      });
```

Comment:
Hey @utnim2, your work looks good overall, but it seems like the dialog box isn't functioning properly. Could you please look into this this? I suspect you might have missed importing the dialog box somewhere.

Reply:
Thanks for your review @Abhijay007 ,looking into it

Reply:
Yes thanks @Abhijay007 for the review!  @utnim2, I think also we're missing the call to ` setUnsavedChanges(true);`, so `unsavedChanges` is always `false`.

Reply:
@lrasmus was working on this only. Pushing some changes in some moments